assets/app/features/upgrade/upgrade.controller.js

```diff
@@ -3,53 +3,31 @@
 
     /**
      * @ngdoc function
-     * @name crowbarApp.controller:Upgrade7Controller
+     * @name crowbarApp.controller:UpgradeController
      * @description
-     * # Upgrade7Controller
+     * # UpgradeController
      * This is the controller that will be used across the upgrade process.
      */
     angular.module('crowbarApp.upgrade')
-        .controller('UpgradeController', Upgrade7Controller);
+        .controller('UpgradeController', UpgradeController);
 
-    Upgrade7Controller.$inject = ['$scope', '$translate', '$state', 'upgradeStepsFactory'];
+    UpgradeController.$inject = ['$scope', '$translate', '$state', 'upgradeStepsFactory'];
     // @ngInject
-    function Upgrade7Controller($scope, $translate, $state, upgradeStepsFactory) {
+    function UpgradeController($scope, $translate, $state, upgradeStepsFactory) {
         var vm = this;
         vm.steps = {
             list: [],
-            activeStep: {},
             nextStep: nextStep,
-            isLastStep: isLastStep
+            isLastStep: isLastStep,
+            stepFinished: upgradeStepsFactory.stepFinished
```

Comment:
Not sure what our convention is but we should try to keep the functions returning booleans named consistently. Here we have `isLastStep()` and `stepFinished()`.